### DISCUSSION TITLE
It seems tool is missing some code or the whole requirement tools

### DISCUSSION THREAD

**Discussion by saifkhalidpersonal27** [HACK3RY2J/Anon-SMS#13] [Q&A]
It shows about the error some dependcis missing

**Comment by Bunny-on-github** [comment]
Theres Some Error in code we will Fix it in future updates.
For now you can reinstall the tool and it will be done

**Reply by sashikiran720** [reply]
It is not working

**Comment by saifkhalidpersonal27** [comment]
Okay
